content/en/agent/logs/_index.md

```diff
@@ -172,6 +172,24 @@ List of all available parameters for log collection:
 
 To send logs over HTTPS with the Datadog Agent 6.14+, add the following in the Agent's [main configuration file][4] (`datadog.yaml`):
 
+{{< tabs >}}
+{{% tab "Compression enabled" %}}
+
+```
+logs_config:
+  use_http: true
+  use_compression: true
+  compression_level: 6
+```
+
+Or set the `DD_LOGS_CONFIG_USE_HTTP` and `DD_LOGS_CONFIG_USE_COMPRESSION` environment variables to `true`.
+The `compression_level` parameter (or `DD_LOGS_CONFIG_COMPRESSION_LEVEL` environment variable) accept value from 0 (no compression) to 9 (maximum compression but higher resource usage). The default value is 6.
+
+Then restart the Agent to sends logs through HTTPS to `agent-http-intake.logs.datadoghq.com` (US site) or `agent-http-intake.logs.datadoghq.eu` (EU site) on port 443.
```

Comment:
is the host name relevant here?  shouldn't setting the  "site" be sufficient?

Reply:
Indeed setting site is sufficient to choose to send data to US or EU site.
The goal of highlighting the domain the agent send the logs to is for users that whitelist this in their firewall policy.